.travis.yml

```diff
@@ -1,3 +1,19 @@
+#
+#  Copyright 2014-2016 CyberVision, Inc.
```

Comment:
Should be changed to `Copyright 2016 CyberVision, Inc.`. There were no this file back in 2014